packages/teleterm/src/ui/services/clusters/clustersService.ts

```diff
@@ -591,8 +594,25 @@ export class ClustersService extends ImmutableStore<ClustersServiceState> {
     return await this.client.listDatabaseUsers(dbUri);
   }
 
+  async removeClusterKubeConfigs(clusterUri: string): Promise<void> {
+    const {
+      params: { rootClusterId },
+    } = routing.parseClusterUri(clusterUri);
+    if (!rootClusterId) {
+      throw new Error(
+        'Could not remove kube configs, `rootClusterId` is missing.'
+      );
+    }
+    return this.mainProcessClient.removeKubeConfig({
+      relativePath: rootClusterId,
+      isDirectory: true,
+    });
+  }
+
   async removeKubeConfig(kubeConfigName: string): Promise<void> {
```

Comment:
It's hard to tell if those two methods belong here or not. Technically they don't have anything to do with the state that ClustersService manages, I could imagine a completely separate KubeConfigService which accomplishes the same thing.

For now it's fine, but let's keep that in mind for this feature and any future ones.